### PR TITLE
Fix fake players not being able to respawn for exiting the end

### LINKED ISSUE #1645: Fake players get stuck in the end exit portal
the same issue as #82 but it happens again in 1.19.2

When a fake player collides with the end exit portal, it becomes invisible, and its position is at the end exit portal.
You can't interact with it, but the /tp command works, and it can make the fake player visible

Carpet version: v1.4.84
Minecraft version: 1.19.2

I haven't tested this in 1.19.3

### PR DESCRIPTION
**Pull request by silnarm**
* Allow fake players to respawn (to return from the end)
* Fixes #1645 (Fake players get stuck in the end exit portal)

### CODE REVIEW

**Comment by altrisi** [comment]
Wondering, what's the reason this is a draft?

File: src/main/java/carpet/patches/EntityPlayerMPFake.java [hunk 1]
```diff
@@ -6,9 +6,7 @@
 import net.minecraft.core.UUIDUtil;
 import net.minecraft.network.chat.Component;
 import net.minecraft.network.protocol.PacketFlow;
-import net.minecraft.network.protocol.game.ClientboundPlayerInfoUpdatePacket;
-import net.minecraft.network.protocol.game.ClientboundRotateHeadPacket;
-import net.minecraft.network.protocol.game.ClientboundTeleportEntityPacket;
+import net.minecraft.network.protocol.game.*;
```
Comment (gnembon) [inline src/main/java/carpet/patches/EntityPlayerMPFake.java:9]: unstar please

**Comment by gnembon** [comment]
same - these is pure gold stuff. Why it is still wip?

**Review by altrisi — APPROVED** [review]

**Comment by altrisi** [comment]
Thanks for contributing!

**Pull request merged**